README.md

```diff
@@ -105,7 +106,71 @@ Then recreate a new, empty namespace:
 ./init-namespace.sh
 ```
 
-### Setup Cluster
+### Setup Vault
+TSI requires Vault to store the secrets. If you have a Vault instance that can be
+used for this TSI installation, make sure you have admin privileges to access it.
+Otherwise, follow the simple steps below to create a Vault instance, as a pod and
+service, deployed in `trusted-identity` namespace in your cluster.
+
+```console
+kk create -f examples/vault-plugin/vault.yaml
+service/tsi-vault created
+deployment.apps/tsi-vault created
+```
+
+#### Obtain remote access to Vault service
```

Comment:
Suggestion: we can do a collapsable for this depending on minikube vs IKS something like this https://github.com/cncf/sig-security#sig-members